lib/screens/login/login_form.dart

```diff
@@ -34,6 +41,10 @@ class _LoginFormState extends State<LoginForm> {
           hintTextStyle: context.textTheme.bodyMedium,
         ),
         style: context.textTheme.bodyMedium,
+        validator: _validatePassword,
+        autovalidateMode: _isFormSubmitted
+            ? AutovalidateMode.onUserInteraction
+            : AutovalidateMode.disabled,
```

Comment:
The cursor color turns `red` when there is an error. Can we still keep the `white` color for it?

<img width=300 src="https://github.com/nimblehq/flutter-ic-khanh-thieu/assets/25881847/301c4792-0f05-4746-ac91-f5c0c63e1865">

Reply:
I noticed this too!
However, it seems that the latest Flutter matches the color of `errorText` with `cursorColor` when the input is invalid. Previous Flutter sdk works though 🙏